Jenkinsfile

```diff
@@ -1,10 +1,20 @@
 pipeline {
-    agent { docker { image 'python:2.7.17' } }
-    stages {
-        stage('build') {
-            steps {
-                sh 'python --version'
-            }
-        }
-    }
+	agent { 
+		docker { image 'python:2.7.17' }
```

Comment:
I feel like this should run the Dockerfile instead, e.g.:

```yaml
    agent { dockerfile true }
```

Reply:
The pipeline isn't doing the containerized deployment of timbot (though this will be added later); the pipeline itself it being run inside of a container